.travis_scripts/build-release.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+# JAVA_HOME isn't set on OSX for some reason, so manually set it
+if [ -z "$JAVA_HOME" ]; then
+    export JAVA_HOME=$(/usr/libexec/java_home)
+fi
+
+cargo b --release
```

Comment:
I don't think this will actually do what you want. This will do a build for release but leave the `--features ironoxide/senv` set. `cargo build --release` should be what's used here.